legal-api/src/legal_api/models/filing.py

```diff
@@ -524,7 +524,9 @@ def json_legal_type(self):
         filing = self._filing_json.get('filing', {})
         legal_type = filing.get('business', {}).get('legalType', None)
         if legal_type is None:
-            legal_type = filing.get('nameRequest', {}).get('legalType', None)
+            legal_type = filing.get('registration', {}).get('nameRequest', {}).get('legalType', None)
+        if legal_type is None:
+            legal_type = filing.get('incorporationApplication', {}).get('nameRequest').get('legalType', None)
```

Comment:
```suggestion
        filing = self._filing_json.get('filing', {})
        legal_type = filing.get('business', {}).get('legalType', None)
        if legal_type is None:
            legal_type = filing.get(self.filing_type, {}).get('nameRequest').get('legalType', None)
```

Reply:
This suggestion was to replace the whole function. Can you remove the duplicate lines 524 and 525